### ISSUE TITLE
Speed up `update-db` tests

### ISSUE THREAD

**Issue by ialarmedalien** [browserslist/browserslist#565]
The `update-db` tests use repos with quite a few dependencies (autoprefixer, postcss-*...). The tests could run more quickly if some less dependency-heavy repos were used -- e.g. [caniuse-on-user-agent](https://www.npmjs.com/package/caniuse-on-user-agent), [caniuse-support](https://www.npmjs.com/package/caniuse-support), [caniuse-api](https://www.npmjs.com/package/caniuse-api), [browserslist-ga](https://www.npmjs.com/package/browserslist-ga). Alternatively, if you want to keep everything "in house", create some npm packages with a bare minimum of content and dependencies (including `caniuse-lite`) and use those.

**Comment by ai** [comment]
I agree. It is a good idea. You can add it to your PR (Sorry, I will not be able to review your PR today).

**Comment by ialarmedalien** [comment]
No problem on not reviewing the PR today. Do you have a preference for which strategy is used -- existing repos vs create testing use only npm modules?

**Comment by ai** [comment]
I think it will be better to use real dependencies
